src/listItem.js

```diff
@@ -0,0 +1,37 @@
+let toDos = JSON.parse(localStorage.getItem('todo-list')) || [];
+
+function listItems() {
+  let li = '';
+  const unorderList = document.querySelector('#ul-list');
+  toDos.forEach((todo, id) => {
+    const toDos = `
+      <li>
+          <div id="list-items">
+            <input type="checkbox" id="${id}"/><label for="${id}">${todo.task}</label>
+          </div>
+          <div id="edit">
+              <i id="ellipsis" class="fa-solid fa-ellipsis"></i>
+              <ul id="delete-box">
+                  <li onclick="editTask(${id}, '${todo.task}')"><i class="fa-solid fa-pen"></i>Edit</li>
+                  <li onclick="deleteTask(${id})"><i class="fa-solid fa-trash"></i>Delete</li>
+              </ul>
+          </div>
+      </li>`;
+    li += toDos;
+  });
+  unorderList.innerHTML = li;
+
+  window.deleteTask = function (deletedId) {
+    toDos.splice(deletedId, 1);
+    toDos = toDos.map((todo, index) => {
+      todo.index = index + 1;
+      return todo;
+    });
+    localStorage.setItem('todo-list', JSON.stringify(toDos));
+    listItems();
+  };
+}
```

Comment:
- [ ] You have done a terrific job so far, but you have not implemented the functionality to remove all items marked as completed at once. This is a core requirement, and I believe it falls within the previous reviewer's suggestion that your Add/Delete is not working as expected. Please fix this 🙏 

**NB:** In order to get this to work, first you need to implement 'mark as completed'. Right now, marking the checkbox has no effect in your app. When I click the checkbox, I expect that task to be marked as completed in the `localStorage`. This will make it easier for you to be able to isolate items that need to be cleared when a user clicks the 'Clear all completed' button.
Happy coding 👨‍💻

Reply:
Hi There,
@codepantha , First of all, thanks for your nice suggestions. Secondly, with due respect, I want to say that this project is for yesterday, not today, the functionality to remove all the items completed at once is required for today.

Waiting for your reply.

Reply:
@safar1212. Thank you for pointing that out. My apologies.
** Note to next reviewer ** Kindly approve 👍